src/frontend/classic/App.svelte

```diff
@@ -66,9 +44,13 @@
     // UI
     let isSideNavOpen = false;
     let isOpen = false;
-    let theme: string;
+    let theme = getSettingDefaultValue(storageKeys.THEME, "hakuneko") as string;
```

Comment:
Just set theme to a svelte/store, it'll be easy to store/modify/subscribe and no need to pass it as prop (see the svelte/store comment further)